discord/ui/select.py

```diff
@@ -30,9 +30,9 @@
 from typing import TYPE_CHECKING, Callable, TypeVar
 
 from ..channel import _threaded_guild_channel_factory
-from ..components import SelectMenu, SelectOption
+from ..components import SelectDefaultValue, SelectMenu, SelectOption
 from ..emoji import AppEmoji, GuildEmoji
-from ..enums import ChannelType, ComponentType
+from ..enums import ChannelType, ComponentType, SelectMenuDefaultValueType
 from ..errors import InvalidArgument
 from ..interactions import Interaction
 from ..member import Member
```

Comment:
The changes i've suggested require importing Object 
```suggestion
from ..components import SelectDefaultValue, SelectMenu, SelectOption
from ..emoji import AppEmoji, GuildEmoji
from ..enums import ChannelType, ComponentType, SelectMenuDefaultValueType
from ..errors import InvalidArgument
from ..interactions import Interaction
from ..member import Member
from ..object import Object
```